common/src/main/java/com/wynntils/services/mapdata/MapAttributesResolver.java

```diff
@@ -66,33 +69,44 @@ private <T> T getAttribute(Function<MapAttributes, Optional<T>> attributeGetter)
         return attributeGetter.apply(DefaultMapAttributes.INSTANCE).get();
     }
 
+    private ResolvedMarkerOptions getResolvedMarkerOptions(
+            Function<MapAttributes, Optional<MarkerOptions>> attributeGetter) {
+        return new ResolvedMarkerOptions(
+                getInheritedValue(MarkerOptions::getInnerRadius, attributeGetter),
+                getInheritedValue(MarkerOptions::getOuterRadius, attributeGetter),
+                getInheritedValue(MarkerOptions::getFade, attributeGetter),
+                getInheritedValue(MarkerOptions::getBeaconColor, attributeGetter),
+                getInheritedValue(MarkerOptions::labelVisible, attributeGetter),
+                getInheritedValue(MarkerOptions::distanceVisible, attributeGetter),
```

Comment:
I'm still not really happy about the names. Let start with the two first. Sure, it technically describes a radius, but that could also just be described as the player's distance from the target, which I think seems more intuitive. So that then points towards something like `getSmallestDistance` and `getLargestDistance`. But that is unwieldy. So maybe just try to and align it with the visibility attribute and call it `getMin` and `getMax`? 

Or... maybe `getMin/MaxDistance`, and rename the visibility attributes `getMinZoom` and `getMaxZoom`, to clarify the difference a bit more?